bareos-setup-vue/src/css/quasar.variables.scss

```
@@ -0,0 +1,11 @@
+$primary   : #0075be;
```

Stylelint (17.7.0)

[error] Unexpected whitespace before ":" (scss/dollar-variable-colon-space-before)

(scss/dollar-variable-colon-space-before)